lib/ASRouter.jsm

```diff
@@ -994,6 +1050,9 @@ class _ASRouter {
         await this.blockProviderById(action.data.id);
         this.messageChannel.sendAsyncMessage(OUTGOING_MESSAGE_NAME, {type: "CLEAR_PROVIDER", data: {id: action.data.id}});
         break;
+      case "DISMISS_BUNDLE":
```

Comment:
I'm leaving `BLOCK_BUNDLE` action in here because tests were complaining about coverage